### ISSUE TITLE
Collector removes link in mail body

### ISSUE THREAD

**Issue by felag** [glpi-project/glpi#13625]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

9.5.11

### Bug description

Mail body contains a link, which is not found in ticket.

### Relevant log output

```shell
Here is mail body (with anonymized information) :

-----Message d'origine-----
De : SG E-brochure <noreply@ebrochure.ma> 
Envoyé : mercredi 14 décembre 2022 16:28
À : ANONYMIZED ADDRESS <anonymized.address@domain.tld>
Objet : E-brochure

Bonjour, 


Veuillez trouver la brochure Société Générale, personnalisée pour votre entreprise. Cette sélection de solutions répond aux besoins identifiés à ce jour pour votre organisation. 

Nous restons à votre disposition pour vous apporter toute éventuelle explication ou information complémentaire vous permettant la concrétisation de vos projets. 

Nous vous remercions de votre confiance et espérons que nos solutions vous apportent entière satisfaction. 

Nous restons mobilisés pour vous accompagner.

Télécharger <https://ebrochure.ma/pdf-viewer/external/pdfjs/web/viewer.php?url=https%3A%2F%2Febrochure.ma%2Ftelecharger%3Fproduits%3D67%26client%3D%257B%2522nom_client%2522%253A%2522ANONYMIZED%2520ADDRESS%2522%252C%2522email_professionnel_client%2522%253A%255B%257B%2522value%2522%253A%2522anonymized.address%2540domain.tld%2522%257D%255D%252C%2522entreprise_client%2522%253A%2522BENTIN%2520ENGINEERING%2522%252C%2522telephone_professionel_client%2522%253A%252207%252064%252042%252095%252025%2522%257D%26contacts%3D%257B%2522nom_bce%2522%253A%2522EL%2520BOURKKADI%2520RAJAA%2522%252C%2522telephone_bce%2522%253A%2522%252B212602155947%2522%252C%2522email_bce%2522%253A%2522rajaa.el-bourkkadi%2540socgen.com%2522%252C%2522selected_caf%2522%253A%25221%2522%257D>  

Cordialement,

Nom conseiller : EL BOURKKADI RAJAA
Email conseiller : rajaa.el-bourkkadi@socgen.com Téléphone conseiller : +212602155947
```

### Your GLPI setup information

GLPI 9.5.11.

### Anything else?

Mail body does not seem to respect standards (`<https…` tag does not exist).

Is this why link has been removed ?

Thanks and Regards

**Comment by cedric-anne** [comment]
Hi,

> Mail body does not seem to respect standards (<https… tag does not exist).

This is probably the problem here. In GLPI 9.5, we had some issues related to handling of `<` and `>` in rich texts.

It is probably fixed in GLPI 10.0. I keep this issue open and will try to check if problem has been fixed.

**Comment by trasher** [comment]
This is fixed at least in GLPI 10.0.6, a dedicated test has been added

**Comment by cedric-anne** [comment]
I just tested it and indeed, in GLPI 10.0.6 it is correctly rendered.

![image](https://user-images.githubusercontent.com/33253653/214593748-ef81ac8e-7ead-45f1-a50b-4dd342dda295.png)
